README.md

```diff
@@ -22,6 +22,7 @@ Once installed, you can type `fab -l` to see a list of available commands.
 ### 1. Build and start Docker containers
 
 ```sh
+cp .env.example .env
```

Comment:
I think this should probably go in the "Setting up a local build" section above as you don't need to do it every time to build and start the containers

Reply:
Actually isn't this part of the instructions for setting up a local build? I read the `### 1. Build and start Docker containers` as a sub-heading of `## Setting up a local build`

Reply:
So it is! 🙃

In that case, I'd probably suggest moving this line above the `### 1. Build and start Docker containers` as a new "First time setup" section.

Reply:
I've done that (although note that there are other items in this longer section which would be first time only, such as creating an account)

Reply:
Probably can move them in as well but I'm marking this thread as resolved 👍🏼